core/src/components/picker-column-option/picker-column-option.ios.scss

```diff
@@ -0,0 +1 @@
+@import "./picker-column-option.scss";
```

Comment:
Our existing components leave off the file extension with importing another SASS file.